src-plugins/vtkDataMesh/readers/vtkDataMeshReader.cpp

```diff
@@ -89,28 +90,124 @@ bool vtkDataMeshReader::read(const QString& path) {
             return false;
         }
 
-        try
-        {
-            dataSet->Read(path.toLocal8Bit().constData());
```

Comment:
Note: i'm testing but it may need to change also vtkDataMeshWriter which have been merged in an  other PR.